packages/InteractiveProfilingTool-Core.package/IPTModel.class/instance/buildButtonWith..st

```diff
@@ -0,0 +1,20 @@
+as yet unclassified
```

Comment:
We should consider classifying the functions

Reply:
Fixed in 5f12b17bfd440baf1a2c1a55b3ac8f7f299815fc